app/src/App/OnDeviceDisplayApp.tsx

```diff
@@ -247,7 +247,7 @@ export const OnDeviceDisplayApp = (): JSX.Element => {
   }, [dispatch, isIdle, usersBrightness])
 
   return (
-    <ApiHostProvider hostname="localhost">
+    <ApiHostProvider hostname="10.13.11.47">
```

Comment:
I do this all the time lol

```suggestion
    <ApiHostProvider hostname="localhost">
```